nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -272,6 +272,11 @@ Available as [services.patroni](options.html#opt-services.patroni.enable).
   apply patches. This means that your hashes are out-of-date and the (previously required) attributes `name` and `version`
   are no longer accepted.
 
+- The Syncthing service now only allows absolute paths---starting with `/` or
+  `~/`---for `services.syncthing.folders.<name>.path`.
+  In a future release relative paths will be allowed again and interpreted
```

Comment:
That sounds weird to me. I don't think we will allow relative paths in a future release: we'll just allow writing a path without a preceding `/` or `~/`, and we'll treat them as relative. Perhaps better wording would be "The Syncthing service now only allows explicit absolute paths, in a future release all implicit paths would be interpreted as `~/<path>`" or something of the like?

Reply:
How about
```suggestion
  In a future release other paths will be allowed again and interpreted
```
?

Reply:
Still a little off, how about `old path format will be allowed again`?

Reply:
> we'll just allow writing a path without a preceding / or ~/, and we'll treat them as relative

Is that not a relative path? The current wording seems fine to me. I've never heard of an "implicit path"

Reply:
I've always thought `../whatever` to be a relative path. `~/whatever` is just an alias to `/home/user/whatever`, which is an absolute path.
Honestly, I feel like this PR is good to be merged regardless of the naming chosen, so choose what you like! I'm just throwing some ideas out there to see which ones stick.